app/models/court.rb

```diff
@@ -0,0 +1,27 @@
+class Court
+  def initialize(name:)
+    @name = name
+  end
+
+  attr_reader :name
+
+  class << self
+    # TODO: we still need to confirm which courts to list.
+    #
+    # In the meantime use the CourtCentre csv from hmcts_common_platform gem
+    # and filters by oucode_l1_code to produce a list of magistrates' and crown courts.
+    #
+    def all
+      @all ||= begin
+        oucode_l1_code = %w[B C]
+
+        rows = HmctsCommonPlatform::Reference::CourtCentre.csv.select do |cc|
```

Comment:
I left a note about this in the PR.  .all does not expose ourcode_l1_code. I've confirmed they'd accept a pr which exposes this. I thought it best to get the find court list specification, before putting that PR in in case we don't actually need it.